### PR TITLE
PrismaCloud - added new command for inventory scan

### PR DESCRIPTION
**Pull request by fvigo**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Added new `!redlock-search-config` command to scan for inventory

## Minimum version of Demisto
- [x] 5.0.0
- [ ] 5.5.0
- [ ] 6.0.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

File: Packs/PrismaCloud/ReleaseNotes/1_3_0.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### Prisma Cloud (RedLock)
+- Added new command: !redlock-search-config
```
Comment (yaakovi) [inline Packs/PrismaCloud/ReleaseNotes/1_3_0.md:4]:
```suggestion
Added the ***redlock-search-config*** command.
```

File: Packs/PrismaCloud/Integrations/RedLock/RedLock.py [hunk 2]
```diff
@@ -415,6 +417,43 @@ def get_remediation_details():
...
+def redlock_search_config():
+    """
+    Run query in config
+    """
+    query = demisto.args().get('query', None)
+    if not query:
+        return_error('You must specify a query to retrieve assets')
+    payload = {
+        'query': query,
+        'limit': 100,
```
Comment (yaakovi) [inline Packs/PrismaCloud/Integrations/RedLock/RedLock.py:429]: should be a command argument

File: Packs/PrismaCloud/Integrations/RedLock/RedLock.py [hunk 3]
```diff
@@ -415,6 +417,43 @@ def get_remediation_details():
...
+        'sort': [{"direction": "desc", "field": "insertTs"}],
+        'withResourceJson': True
+    }
+    handle_time_filter(payload, {'type': 'to_now', 'value': 'epoch'})
```
Comment (yaakovi) [inline Packs/PrismaCloud/Integrations/RedLock/RedLock.py:433]: should be a command argument

Comment (fvigo) [reply Packs/PrismaCloud/Integrations/RedLock/RedLock.py:441]: @yaakovi not exactly sure what you mean here, the command uses the filters provided as imputs

**Review by yaakovi — APPROVED** [review]

**Comment by yaakovi** [comment]
@ShirleyDenkberg @kirbles19 
please review the new command in the yml starting from line 494.

**Comment by ShirleyDenkberg** [comment]
@yaakovi Doc review completed.

File: Packs/PrismaCloud/Integrations/RedLock/RedLock.yml [hunk 4]
```diff
@@ -494,11 +491,99 @@ script:
...
-      description: The alert ID for which to get remediation details.
+  - arguments:
+    - description: Start time for the search, in the following string format -  MM/DD/YYYY
+      name: time-range-date-from
+    - description: End time for the search, in the following format -  MM/DD/YYYY
+      name: time-range-date-to
+    - description: The number of time range value units for the search. For example, 3 days, 5 weeks, etc.
+      name: time-range-value
+    - auto: PREDEFINED
+      description: The search unit. Possible values are: "hour", "week", "month", "year", "login", or "epoch". The "login" and "epoch" options are only available if timeRangeValue
```
Comment (yaakovi) [inline Packs/PrismaCloud/Integrations/RedLock/RedLock.yml:502]:
```suggestion
      description: |-
        The search unit. Possible values are: "hour", "week", "month", "year", "login", or "epoch". The "login" and "epoch" options are only available if timeRangeValue
```

**Pull request merged**